JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -92,14 +93,15 @@ module Microsoft.ApplicationInsights {
                 this.session = new Context.Session();
                 this.sample = new Context.Sample(config.sampleRate());
             }
+
+            this._addDefaultTelemetryInitializers();
```

Comment:
Yeah, I thought about it and decided that instrumenting browser link is not useful and I didn't want to add another config field (we already have plenty).
But you're right, somebody with similar routes may want to disable it. I'll add it.